src/modules/endpoint/models/__tests__/DateModel.spec.js

```diff
@@ -0,0 +1,95 @@
+import DateModel from '../DateModel'
+import moment from 'moment'
+
+describe('DateModel', () => {
```

Comment:
Maybe we should include tests for other locales as well

Reply:
will be much effort, especially because I'm not able to speak arabic/farsi

Reply:
Maybe u can do sth lik 
```javascript
locales = ['de', 'fa', 'ru', 'ar', 'en']
expect(locales.map(locale => dateModel.toLocaleString(locale))).toMatchSnapshot()
```
and then just verify real quick with Google Translate or similar (not even necessary imo). Import thing is that it actually does something in other languages than German, and if it's doing the right format in German, other languages should be correct I think.

Reply:
yeah sounds good, will be done

Reply:
`dateModel.toLocaleString` no longer exists. This is now part of TimeSpan

Reply:
Is there a reason why `toLocaleString` is again part of DateModel and Timespan is gone?
toLocaleString() *renders* a combination of Dates so why not make a component of it?

This is a lot of unnecessary complexity for a model.

Reply:
if you wish I could move it to a class TimeSpan again, just thought as it is only one function and it fits logically into DateModel it would be ok, so maybe we should rather rename DateModel to TimeSpan because it's not really a DateModel

Reply:
Yeah, TimespanModel for the model and Timespan for the comoponent (or view). Sounds reasonable 👍

Reply:
I removed DateModel completely and added it's getter/setter functionality to EventModel, and created a TimeSpan component for the toTimeSpanString() functionality